website/docs/docs/dbt-cloud-apis/access-metadata-api.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Access the API"
+id: "access-metadata-api"
+---
+
+Use the Metadata API to evaluate data health in the long-term or at a moment-in-time. This section will explain how to access and query the Metadata API and what you'll need.
+
+<Snippet src="metadata-api-prerequisites" />
+
+We provide a [graphical explorer](https://metadata.cloud.getdbt.com/graphql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query. 
```

Comment:
Should this say "We provide a GraphQL explorer..." since we're using a product name?

Reply:
we can! this was existing content and i haven't rewritten the existing docs but I'll update that